src/vm/jitinterface.cpp

```diff
@@ -1302,9 +1302,23 @@ struct TryResolveTokenFilterParam
 
 bool isValidTokenForTryResolveToken(CEEInfo* info, CORINFO_RESOLVED_TOKEN* resolvedToken)
 {
-    LIMITED_METHOD_CONTRACT;
+    CONTRACTL {
+        NOTHROW;
+        GC_TRIGGERS;
```

Comment:
Is there any way to call `isValidToken` while ensuring that we do not trigger a GC?